BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -485,14 +473,14 @@ public bool AutoExposureColor
         {
             get
             {
-                CheckOptionSupported(RealSense2API.Option.ENABLE_AUTO_EXPOSURE, AutoExposureColorDesc.Name, RealSense2API.SensorName.COLOR);
-                return RealSense2API.GetOption(_pipeline, RealSense2API.SensorName.COLOR, RealSense2API.Option.ENABLE_AUTO_EXPOSURE) == 1.0f ? true : false;
+                CheckOptionSupported(Option.AutoExposurePriority, AutoExposureColorDesc.Name, SensorName.COLOR);
```

Comment:
This looks like an error.
* The option name changed from `ENABLE_AUTO_EXPOSURE` to `AutoExposurePriority`
* The param below (`AutoExposurePriorityColor`) is identical to this one

Reply:
Yes, should have been `EnableAutoExposure`. Probably c&p mistake.

Reply:
I'm always delighted to find those :-)